tests/scaffoldings/dotd/custom/extras/goodconf.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/local/bin/python-odoo-shell
+from odoo.tools import config
+assert config.get("limit_request") == "8000"
```

Comment:
This still [fails on line 3](https://travis-ci.org/Tecnativa/docker-odoo-base/jobs/327804781#L5232-L5233).

I'm suspecting that duplicate keys cannot be correctly parsed, or that the parameter is converted to `int` and thus this assertion fails.

Maybe could you add assertion messages that help us to debug further? Example:

```python
assert config.get("limit_request") == 8000, "%r is not %r" % (config.get("limit_request"), 8000)
```

... and the same in line 4